proto/src/build.rs

```diff
@@ -2,6 +2,10 @@ use std::io::Result;
 
 fn main() -> Result<()> {
     std::env::set_var("PROTOC", protobuf_src::protoc());
-    prost_build::compile_protos(&["proto/test_context.proto"], &["proto/"])?;
+    let mut prost_build = prost_build::Config::new();
+    prost_build
+        .type_attribute(".", "#[derive(serde::Serialize,serde::Deserialize)]")
+        .extern_path(".google.protobuf.Timestamp", "::prost_wkt_types::Timestamp")
+        .compile_protos(&["proto/test_context.proto"], &["proto/"])?;
```

Comment:
This adds serialization / deserialization support. `Prost` recommends to override the google types with their own in order to use serialization.

Reply:
Had to do the same for the bazel-bep stuff. I started work to try to unify these more but hit a bit of a wall: https://github.com/trunk-io/analytics-cli/pull/208